bindata/scripts/enable-kargs.sh

```diff
@@ -12,6 +14,12 @@ if grep --quiet CoreOS "$REDHAT_RELEASE_FILE"; then
         fi
     done
 else
+    chroot /host/ which grubby
```

Comment:
Shall we redirect the stdout and stderr for the `which` cmd? otherwise it may appear in the exit code and break the check condition.

Reply:
Oh, you are right. I just tested (my) happy path where which was failing.